vrc-oscquery-lib/Zeroconf/MeaModDiscovery.cs

```diff
@@ -23,12 +23,12 @@ public class MeaModDiscovery : IDiscovery
 
         public void Dispose()
         {
-            if (_discovery != null)
-            {
-                foreach (var profile in _profiles.Keys.ToArray())
-                    Unadvertise(profile);
-
-                _discovery.Dispose();
+            if (_discovery != null)
+            {
+                foreach (var profile in _profiles.Keys.ToArray())
+                    Unadvertise(profile);
+
+                _discovery.Dispose();
```

Comment:
idk what these are, seems like Rider fixed the indentation but I don't see any difference. Maybe I'm blind

Reply:
Ok, I force-pushed and this time I edited by hand, it seems this seems to be a git merge commit thing lol

Reply:
hiding whitespace hides these, if it's not indentation then it might be git modifying the line endings on account of weird git configuration

Reply:
yeah, you probably right. I'm in Linux and this repository probably uses LF only, right? 

I have `core.autocrlf=input`

Reply:
```
$ file vrc-oscquery-lib/vrc-oscquery-lib/Zeroconf/ZeroconfTypes.cs
vrc-oscquery-lib/vrc-oscquery-lib/Zeroconf/ZeroconfTypes.cs: Unicode text, UTF-8 (with BOM) text, with CRLF line terminators
$ file vrc-oscquery-lib/vrc-oscquery-lib/Zeroconf/MeaModDiscovery.cs
vrc-oscquery-lib/vrc-oscquery-lib/Zeroconf/MeaModDiscovery.cs: Unicode text, UTF-8 (with BOM) text, with CRLF, LF line terminators
```
looks like it might not be standardized in the file anyways. Hmm...

Reply:
Well, it should be solved in another PR. And probably added inside the `.gitattributes`